substrate/frame/system/src/lib.rs

```diff
@@ -628,6 +636,28 @@ pub mod pallet {
 			Self::deposit_event(Event::Remarked { sender: who, hash });
 			Ok(().into())
 		}
+
+		#[pallet::call_index(8)]
+		#[pallet::weight(task.weight())]
+		pub fn do_task(origin: OriginFor<T>, task: T::RuntimeTask) -> DispatchResultWithPostInfo {
```

Comment:
So we are not calling this from anywhere yet, or?

Reply:
Correct - just an extrinsic for now.